skins/shared/scripts/onScroll.js

```diff
@@ -45,7 +45,7 @@ define('wikia.onScroll', ['jquery', 'wikia.window'], function ($, window) {
 	function init() {
 		window.addEventListener(
 			'scroll',
-			$.debounce(debounceRate, true, trigger)
+			$.throttle(delay, trigger)
```

Comment:
Let's make sure to test other features that use `wikia.onScroll` before merging this.